.ci/pipeline/ci.yml

```diff
@@ -46,7 +46,45 @@ jobs:
       targetPath: '$(release.dir)'
   - script: |
       .ci/scripts/test.sh --test-kind examples --build-dir $(release.dir) --platform $(platform.type)
-    displayName: 'examples'  
+    displayName: 'examples'
+- job: daal4py
+  dependsOn: Linux
+  timeoutInMinutes: 0
+  variables:
+    release.dir: '__release_lnx_gnu'
+  pool:
+    vmImage: 'ubuntu-18.04'
+  strategy:
+    matrix:
```

Comment:
you can remove strategy section - it's intended for iterating via several configuration sets